enzy_htp/_config/alphafill_config.py

```diff
@@ -29,6 +29,12 @@ class AlphaFillConfig(BaseConfig):
     MAXIT:str="maxit"
     """Path to the maxit executable for converting .pdb files to mmCIF."""
 
+    PDB_FASTA:str=""
+    """Path to the FASTA file for the master pdb database."""
```

Comment:
What is the "master" pdb? If it is coupled with specific use cases and will be the info that vary everytime user call an API, it should be part of the arguments of that API. (same form PDB_DIR)